lib/logstash/inputs/logstash.rb

```diff
@@ -32,7 +32,6 @@ class LogStash::Inputs::Logstash < LogStash::Inputs::Base
 
   # SSL:TRUST:CONFIG
   config :ssl_client_authentication,   :validate => %w(none optional required), :default => 'none'
-  config :ssl_verification_mode,       :validate => %w(certificate),            :default => 'certificate'
```

Comment:
removing SSL verification mode from input in favor of `ssl_client_authentication`, source: https://github.com/logstash-plugins/logstash-input-http/pull/165/files